input/pagecontent/2-5-EventNotification.md

```diff
@@ -113,41 +94,39 @@ If the subscriber cannot follow the context of the event, for instance due to an
 
 If the Hub receives an error notification from a subscriber, it SHALL generate a [`syncerror`](3-2-1-syncerror.html) event to the other subscribers of that topic. [`syncerror`](3-2-1-syncerror.html) events are like other events in that they need to be subscribed to in order for an app to receive the notifications and they have the same structure as other events, the context being a single FHIR `OperationOutcome` resource.
 
-The figure below illustrates the `webhook` and `websocket` Event Notification Error Sequence.
+The figure below illustrates the Event Notification Error Sequence.
 
-{% include img.html img="ErrorSequence.png" caption="Figure: Event Notification Error flow diagram" %}
+{%include EventNotificationErrorSequence.svg%}

```

Comment:
Should sequence diagram also show event and sync error being sent to the "Driving App"?  Also, "Drving App" should be "Driving App" unless intentionally made shorter.